tests/StringViewTest.cpp

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+#include <aws/crt/Api.h>
+#include <aws/crt/StringView.h>
+#include <aws/crt/Types.h>
+#include <aws/testing/aws_test_harness.h>
+
+static int s_test_string_view(struct aws_allocator *allocator, void *ctx)
+{
+    (void)ctx;
+    Aws::Crt::ApiHandle apiHandle(allocator);
+
+    {
+        const char *data = "abc123xyz";
+
+        Aws::Crt::StringView sv(data);
```

Comment:
While these don't need to be split up as individual tests, it would be nice if they were at least broken up by subject/theme.